Botbases/RSBot.Default/Subscriber/BundleSubscriber.cs

```diff
@@ -0,0 +1,41 @@
+﻿using RSBot.Core.Event;
+using RSBot.Default.Bundle;
+
+namespace RSBot.Default.Subscriber;
+
+internal class BundleSubscriber
+{
+    public static void SubscribeEvents()
+    {
+        EventManager.SubscribeEvent("Bundle.Loop.Start", Bundles.Loop.Start);
```

Comment:
We will need allmost all those functions in a trade bot, too, that's why I decided to solve the dependency problem this way instead of having duplicated code in every botbase. A second method would be to move every bundle to the core but I don't like that. Now we can reuse the functions easily without dependency hell.